src/server/scripts/Custom/SkipDK.cpp

```diff
@@ -62,7 +62,7 @@ class Trinitycore_skip_deathknight : public PlayerScript
 
         if (sConfigMgr->GetBoolDefault("Skip.Deathknight.Starter.Enable", true))
         {
-            if (player->GetAreaId() == 4342)
+            if (player->GetSession()->GetSecurity() == SEC_PLAYER && player->GetAreaId() == 4342)
```

Comment:
there are IsGameMaster() (not set yet on login) and CanBeGameMaster() (checks if player can use ".gm on" command), any of these that could be used instead ?

Reply:
in this case this is for the abilitiy to have it specific with just normal player accounts already and the other bool down in line 121 is speific for GM level accounts of moderator or higher. the issue reported was the people who have the full skip on for both players and gms, because i am not specifiying player accounts alone and seperate from the gm accounts, the gm account ends up with dupes of all items in the module which leads to errors on the console being reported.